subways/tests/test_route.py

```diff
@@ -0,0 +1,140 @@
+from unittest import TestCase
+
+from subways.structure.route import (
+    get_interval_in_seconds_from_tags,
+    osm_interval_to_seconds,
+    parse_time_range,
+)
+
+
+class TestTimeIntervalsParsing(TestCase):
+    def test__osm_interval_to_seconds__invalid_value(self) -> None:
+        intervals = (
+            ["", "abc", "x30", "30x", "3x0"]
+            + ["5:", ":5", "01:05:", ":01:05", "01:01:00:", ":01:01:00"]
+            + ["01x:05", "01:x5", "x5:01:00", "01:0x:00", "01:01:x"]
+            + ["-5", "01:-05", "-01:05", "-01:00:00", "01:-01:00", "01:01:-01"]
+            + ["0", "00:00", "00:00:00"]
+            + ["00:60", "01:00:60", "01:60:00"]
```

Comment:
technically these cases `["00:60", "01:00:60", "01:60:00"]` may be sufficient, I'd personally add
```
+ ["00:61", "01:00:61", "01:61:00"]
+ ["01:60:61", "01:61:60", "01:61:61"]
```

Reply:
Tests were added, thanks.